jarviscli/plugins/wordgame.py

```diff
@@ -0,0 +1,111 @@
+from plugin import plugin
+import random
+from tkinter import *
+from tkinter import messagebox 
+
+@plugin("wordgame")
+with open("./Jarvis/jarviscli/data/wordgame_word_list.txt", "r") as wordslist:
+    data = wordslist.read()
+    words = data.split()
+    word_pos = random.randint(0, len(words) - 1)
+    random_word = words[word_pos]
+
+root = Tk()
+
+GREEN = "#039108"
+YELLOW = "#fcba03"
+BLACK = "#000000"
+GRAY = "#828282"
+WHITE = "#ffffff"
+
+root.title('Word Game')
+root.geometry('300x400')
+root.resizable(False, False)
+root.config(bg = BLACK)
+
+number_of_guesses = 0
+maximum_trial = 5
+
+word_input = Entry(root)
+word_input.grid(row = 9, column = 0, padx = 10, pady = 10, columnspan = 5)
+word_guessed = []
+
+characters_left = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k',
+                   'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
+                   'w', 'x', 'y', 'z']
+
+
+def get_guess():
+
+    global random_word
```

Comment:
You don't need to 'global' variables if you don' write them. In fact, you should avoid writing global variables.